go.mod

```diff
@@ -36,4 +36,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20211124102406-5f7ca32a57fa
```

Comment:
reminder

Reply:
taken care of in https://github.com/codeready-toolchain/registration-service/pull/231/commits/b98acf7d3774c9ac41306bc8343744a1c5081264